src/utilities/vizProtobuffer/vizMessage.proto

```diff
@@ -319,6 +319,7 @@ message VizMessage{
         double spacecraftHelioViewSizeMultiplier = 49; // Control the display size of spacecraft in the Solar System View, values greater than 0, use negative value to use viz default
         int32 forceStartAtSpacecraftLocalView = 50; // Require Vizard to start up in spacecraft-view on start up
         int32 useSimpleLocationMarkers = 51; // Value of 0 (protobuffer default) to use simplified Location markers when number of locations is greater than 100, -1 to force use of full-featured Location, 1 to force use of simplified Location (no cones, range, or communication lines)
+	    int64 messageBufferSize = 53; //[bytes] Maximum size of vizMessages to be loaded into memory at one time, -1 to force loading of entire file into memory, 0 to use viz default
```

Comment:
These have to be unique, not incremental.  If we used 52 in an earlier version of Vizard and retired that slot as the feature changes, then we don't reuse that slot to avoid potential conflicts with earlier versions of Vizard.